File: program/program/find-surface-area-of-a-cuboid/find_surface_area_of_a_cuboid.pl
print "Enter the Length: ";
my $length = <STDIN> ;

print "Enter the Width: ";
my $width = <STDIN> ;

print "Enter the Height: ";
my $height = <STDIN> ;

my $res = surfaceArea($principal, $interest);
print "Surface area of cuboid: $res";

sub surfaceArea
{
    my ($x) = @_;
    my $res = 2 * ($length * $width +  $length * $height + $width * $height);
    return $res ;   
}

File: program/program/find-surface-area-of-a-cuboid/find_surface_area_of_a_cuboid.php
<?php

echo "Length: ";
fscanf(STDIN, "%f", $length);

echo "Width: ";
fscanf(STDIN, "%f", $width);

echo "Height: ";
fscanf(STDIN, "%f", $height);

$surfaceArea = 2 * (($length * $width) + ($length * $height) + ($width * $height));

echo "Surface Area of Cuboid: " .round($surfaceArea, 2). "\n";

?>